static/tsconfig.json

```diff
@@ -21,6 +21,7 @@
 		"panes/pane.interfaces.ts",
 		"panes/rustmacroexp-view.ts",
 		"panes/rustmir-view.ts",
+		"panes/gnatdebug-view.ts",
```

Comment:
If/when you pull and merge `main` you'll get a conflict here - but it should be an easy one - we've deleted the whole `files` list, which actually fives us the behaviour we need anyway.